selfdrive/thermald/thermald.py

```diff
@@ -447,6 +447,8 @@ def thermald_thread(end_event, hw_queue) -> None:
         except Exception:
           cloudlog.exception("failed to save offroad status")
 
+    params.put_bool_nonblocking("NetworkMetered", (msg.deviceState.networkType != NetworkType.wifi))
```

Comment:
I believe networkmanager tells us over DBus, Android WiFi hotspots for example are marked as metered currently and can't request upload, we should be consistent.